content/en/docs/concepts/deployment-models/index.md

```diff
@@ -0,0 +1,421 @@
+    alt="A service mesh with a single network"
+    title="Single network"
+    caption="A service mesh with a single network"
+    >}}
+
+### Multiple networks
+
+You can span a single service mesh across multiple networks; such a
+configuration is known as **multi-network**.
+
+Multiple networks afford the following capabilities beyond that of single networks:
+
+- An existing network topology
+- Overlapping IP or VIP ranges for **service endpoints**
+- Crossing of administrative boundaries
+- Fault tolerance
+- Scaling of network addresses
+
+In this model, the workload instances in different networks can only reach each
+other through one or more [Istio gateways](/docs/concepts/traffic-management/#gateways).
+Istio uses **partitioned service discovery** to provide consumers a different
+view of {{< gloss >}}service endpoint{{< /gloss >}}s. The view depends on the
+network of the consumers.
+
+{{< image width="75%"
+    link="./multi-net.svg"
+    alt="A service mesh with multiple networks"
+    title="Multi-network deployment"
+    caption="A service mesh with multiple networks"
+    >}}
+
+## Mesh models
+
+Istio supports two mesh models:
+
+- Single mesh
+- Multiple meshes, also known as **multi-mesh**
+
+### Single mesh
+
+The simplest Istio deployment is a single mesh. Within a mesh, service names are
+unique. For example, only one service can have the name `foo/mysvc`.
+Additionally, workload instances all share a common identity and authentication
+infrastructure and can authenticate each other. With a single mesh you can do
+the following tasks:
+
+- Span [one or more clusters](#cluster-models)
+- Span [one or more networks](#network-models)
+- Use [namespaces for tenancy](#namespace-tenancy)
+
+### Multiple Meshes
+
+You can deploy multiple meshes; such a configuration is known as
+**multi-mesh**.
+
+Multiple meshes afford the following capabilities beyond that of a single mesh:
+
+- Organizational boundaries: lines of business
+- Service name or namespace collisions: multiple distinct uses of the `default`
+  namespace
+- Stronger isolation: isolating test workloads from production workloads
+
+In the most basic multiple mesh model, each mesh forms a strict
+boundary and there is no communication between meshes. However, you can enable
+inter-mesh communication with **mesh peering**. When
+peering, each mesh can export a set of services and identities, which all
+participating meshes can recognize.
+
+{{< image width="75%"
+    link="./multi-mesh.svg"
+    alt="Multiple service meshes"
+    title="Multi-mesh"
+    caption="Multiple service meshes"
+    >}}
+
+To avoid service naming collisions, you can give each mesh a globally unique
+**mesh ID**, to ensure that the fully qualified domain
+name (FQDN) for each service is distinct.
+
+When peering two meshes that do not share the same
+{{< gloss >}}trust domain{{< /gloss >}}, {{< gloss >}}identity{{< /gloss >}} and
+**trust bundles** must be federated between them. See the section
```

Comment:
"federated" should point to the glossary entry for mesh federation

Reply:
@frankbu What would be an appropriate definition of `mesh federation` in your opinion?

Reply:
Like I said in my [other comment](https://github.com/istio/istio.io/pull/4813#discussion_r316310594) I think it's the verb form of what I was proposing in https://github.com/istio/istio.io/pull/4788. That is, setting up the loose coupling between the isolated service meshes in a multi-mesh deployment.

It does make we wonder if "peering" and "federating" are the same thing.

@nmittler, WDYT?

Reply:
Yup ... peering and federation are the same thing. Let's replace `peering` with `federation` throughout.